CHANGELOG.md

```diff
@@ -22,6 +22,21 @@ CHANGELOG
 Swift 4.1
 ---------
 
+* [SE-0186][]
+
+  It is no longer valid to use the ownership keywords `weak` and `unowned` for property declarations in protocols. These keywords are meaningless and misleading when used in a protocol as they don't have any effect.
+
+  In Swift 3 and 4 mode the following example will produce a warning with a fixit to remove the keyword. In Swift 5 mode and above an error will be produced.
```

Comment:
Nitpick: "fix-it"

Reply:
Thanks @jrose-apple , I've... fixed-it. :)

The changelog file is inconsistent with spelling and currently has 'fix-it', 'Fix-it' and 'Fix-It'. Any preference? I'll clean it up in a separate pull request if one style is preferred.

Reply:
@natecook1000 should check me on this, but I believe the correct spelling from Apple is "fix-it" (and "Fix-it" at the beginning of a sentence).